NEWS.md

```diff
@@ -12,7 +12,13 @@
 
 ### Changed
 
-* Rename `binary` in `binary_outcome`
+* Add simulation experiments and corresponding visualizations.
```

Comment:
Most of these bullet points are new features. They should be under `Add` section not the Changed section.